bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/TeslaChannelSelectorProxy.java

```diff
@@ -366,18 +366,6 @@ public State getState(String s, TeslaChannelSelectorProxy proxy, Map<String, Str
                 return super.getState(s);
             }
         },
-        FORCE_REFRESH(null, "forcerefresh", OnOffType.class, false) {
```

Comment:
I had a comment on this i deleted, which i should have just updated, please ignore it as i had confused this for the WAKE command when first looking at it.   For reference, the matching channel and command for this ENUM was removed during @kaikreuzer 's refactoring.